endpoints/dev.yaml

```diff
@@ -18,6 +18,7 @@ extraScrapeConfigs: |
         - https://streams.dev.internal.smartcolumbusos.com/socket/healthcheck
         - https://www.dev-smartos.com
         - https://discovery.dev.internal.smartcolumbusos.com
+        - https://data.dev.internal.smartcolumbusos.com
```

Comment:
This returns a 404, btw. Do you maybe want to hit the healthcheck endpoint?

`/api/v1/healthcheck`

Reply:
oh good catch. Thank you!